composer.json

```diff
@@ -42,7 +42,7 @@
         "drupal/ctools": "^4.0.0",
         "drupal/devel": "^5.1.2",
         "drupal/embed": "^1.7",
-        "drupal/entity_browser": "^2.8",
+        "drupal/entity_browser": "^2.9",
```

Comment:
lump this into the commit with embed and entity embed and add a note to that issue.